rest/route/service.go

```diff
@@ -125,6 +125,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}/test_stats").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeGetProjectTestStats(sc))
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(checkUser, viewTasks).RouteHandler(makeGetProjectTaskStats(sc))
 	app.AddRoute("/projects/{project_id}/task_reliability").Version(2).Get().Wrap(checkUser).RouteHandler(makeGetProjectTaskReliability(sc))
+	app.AddRoute("/permissions").Version(2).Get().RouteHandler(&permissionsGetHandler{})
```

Comment:
should we `checkUser`? 
Are we not checking the user because we anticipate not providing a user for MANA?

Reply:
this is all information that's defined in the source code, so it's not particularly private.